c/include/ml-api-common.h

```diff
@@ -98,6 +140,20 @@ typedef enum _ml_tensor_type_e
   ML_TENSOR_TYPE_UNKNOWN         /**< Unknown type */
 } ml_tensor_type_e;
 
+/**
+ * @brief Callback to execute the custom-easy filter in NNStreamer pipelines.
+ * @details Note that if ml_custom_easy_invoke_cb() returns negative error values, the constructed pipeline does not work properly anymore.
+ *          So developers should release the pipeline handle and recreate it again.
+ * @since_tizen 6.0
+ * @remarks The @a in can be used only in the callback. To use outside, make a copy.
+ * @remarks The @a out can be used only in the callback. To use outside, make a copy.
+ * @param[in] in The handle of the tensor input (a single frame. tensor/tensors).
+ * @param[out] out The handle of the tensor output to be filled (a single frame. tensor/tensors).
+ * @param[in,out] user_data User application's private data.
+ * @return @c 0 on success. @c 1 to ignore the input data. Otherwise a negative error value.
+ */
+typedef int (*ml_custom_easy_invoke_cb) (const ml_tensors_data_h in, ml_tensors_data_h out, void *user_data);
```

Comment:
custom-easy is now used only in pipeline, so I think this is not a common callback. declaration.

Reply:
ML-Service-API may register a custom-easy function.